deploy/docker/Dockerfile

```diff
@@ -67,10 +56,19 @@ RUN mkdir /tmp/terraform && \
     chmod +x /bin/terraform && \
     rm -rf /tmp/terraform
 
+# Start with fresh image
+FROM fluent/fluentd:v1.11.1-debian-1.0
+
+USER root
```

Comment:
I wonder if we could drop the `root` user afer all of the apt-gets, mkdirs and chowns.

Reply:
We don't run as root, we run the image as the fluent user.

Reply:
Ah, I see :) 
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/775/files#diff-aa72c1ddeec59a31f30834d956165473R78